### ISSUE TITLE
OpenToonz won't open. What am I doing wrong?

### ISSUE THREAD

**Issue by Grey-Cloud** [opentoonz/opentoonz#3445]
I recently tried building the program with the command line in MacOS 10.14 and it finished, but when I tried opening it I got "LSOpenURLsWithRole() failed with error -10810".

**Comment by artisteacher** [comment]
It looks like sometimes that is a permissions issue?

[https://superuser.com/questions/1564451/macos-catalina-fails-to-open-certain-apps-with-lsopenurlswithrole-failed-with](https://superuser.com/questions/1564451/macos-catalina-fails-to-open-certain-apps-with-lsopenurlswithrole-failed-with)

**Comment by Grey-Cloud** [comment]
Thanks for responding.
I did `chmod +x` to OpenToonz.app and OpenToonz.app/Contents/MacOS/OpenToonz, but the same error still pops up.

**Comment by artisteacher** [comment]
Do you have the same problem if you build with Xcode?

**Comment by Grey-Cloud** [comment]
No. It seems reinstalling Xcode fixed the problem. Although, even though that solved that I'm getting a new issue now:
```
GL_INVALID_FRAMEBUFFER_OPERATION: framebuffer:36053
freeglut  ERROR:  Function <glutStrokeCharacter> called without first calling 'glutInit'.
```
